relay-dynamic-config/src/defaults.rs

```diff
@@ -42,7 +82,7 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
             category: DataCategory::Span,
             mri: "d:spans/exclusive_time_light@millisecond".into(),
             field: Some("span.exclusive_time".into()),
-            condition: None,
+            condition: span_op_conditions.clone(),
```

Comment:
I think this clone should be redundant, since it's the last reference of this struct and can be moved.

```suggestion
            condition: span_op_conditions,
```